### DISCUSSION TITLE
Multiple validations apply only the last message

### DISCUSSION THREAD

**Discussion by Geovanek** [livewire/livewire#7004] [7. Bugs (archived)]
### Livewire version

V3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

```php
#[Rule('required', message: 'Obrigatório')]
#[Rule('email', message: 'Forneça um email válido')]
#[Rule('unique:users,email', message: 'Este e-mail já esta em uso ou você já possui cadastrado na plataforma.')]
public string $email;
```

`message` error always return the unique validation.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by minhtamwebmaster** [comment]
I've encountered a similar issue as well.

**Comment by sueboy19** [comment]
Same problem for me.

Try to use 

```php
#[Rule([
        'chers.*.SubjectID' => 'required',
        'chers.*.det_gold' => 'exclude_unless:chers.*.len_gold,0|required|numeric|min:1',
        'chers.*.len_gold' => 'exclude_unless:chers.*.det_gold,0|required|numeric|min:1',
    ], message: [
        'chers.*.SubjectID.required' => '第:position筆的:attribute空值. 請輸入',
        'numeric' => '請輸入第:position筆的:attribute 必需是數字',
        'min' => '第:position筆 的 :attribute 太少，至少為 :min ',
    ], attribute: [
        'chers.*.SubjectID' => '角色',
        'chers.*.det_gold' => '上金額',
        'chers.*.len_gold' => '下金額',
    ])]
    public $chers;
```

**Comment by PhiloNL** [comment]
Which version are you using? I think this was fixed in https://github.com/livewire/livewire/pull/6761

**Reply by minhtamwebmaster** [reply]
OK  it has been bug-fixed now.
